Sometimes it's possible that the debug info gets disabled
due to alignment issues. If we then get a corruption we
have very few clues wtf happened.

--- linux-2.6.11/mm/slab.c~	2005-04-20 03:07:42.000000000 -0400
+++ linux-2.6.11/mm/slab.c	2005-04-20 03:07:48.000000000 -0400
@@ -1064,6 +1064,7 @@ static void check_poison_obj(kmem_cache_
 				printk(KERN_ERR "Slab corruption: (%s) start=%p, len=%d\n",
 						print_tainted(), realobj, size);
 				print_objinfo(cachep, objp, 0);
+				dump_stack();
 			}
 			/* Hexdump the affected line */
 			i = (i/16)*16;
